### DISCUSSION TITLE
希望支持从百度网盘选择或转存附件的能力

### DISCUSSION THREAD

**Discussion by kwinHou** [halo-dev/halo#5795] [Ideas]
### 你当前使用的版本

2.14

### 描述一下此特性

百度网盘有个人的大量图片、文档啥的，希望能够直接从百度网盘选择附件，而不用把附件先上传到本地附件库

**Comment by LIlGG** [comment]
可以使用插件来实现这样的功能，例如调用百度网盘的 API 。由于这不是一个普适性的功能，我将转移至 Discussion ，你可以关注是否有社区用户帮助编写这样的插件。
